### ISSUE TITLE
Test suite missing modules

### ISSUE THREAD

**Issue by snoyberg** [haskell-servant/servant#68]
```
Resolving dependencies...
Configuring servant-client-0.4.0...
Building servant-client-0.4.0...
Preprocessing library servant-client-0.4.0...
[1 of 3] Compiling Servant.Common.BaseUrl ( src/Servant/Common/BaseUrl.hs, dist/build/Servant/Common/BaseUrl.o )
[2 of 3] Compiling Servant.Common.Req ( src/Servant/Common/Req.hs, dist/build/Servant/Common/Req.o )
[3 of 3] Compiling Servant.Client   ( src/Servant/Client.hs, dist/build/Servant/Client.o )
In-place registering servant-client-0.4.0...
Preprocessing test suite 'spec' for servant-client-0.4.0...

test/Spec.hs:1:8:
    Could not find module ‘Servant.ClientSpec’
    Perhaps you meant
      Servant.Client (needs flag -package-key servant-client-0.4.0@serva_JSTqdDtJ1hWEvU0Dj4ZDIR)
      Servant.Client (from servant-client-0.4.0@serva_6VXrsrfyUXEJrNxZS1Cd6A)
    Use -v to see a list of the files searched for.
```

**Comment by peti** [comment]
We have the same problem in NixOS: http://hydra.cryp.to/build/806518/log/raw.

**Comment by jkarni** [comment]
Thanks for reporting - we clearly should be doing the `cabal sdist` dance in `.travis.yml`. I'll fix this tonight.

**Comment by jkarni** [comment]
Fixed in #96
